transactions/src/main/java/org/ehcache/transactions/xa/internal/XAStore.java

```diff
@@ -745,7 +745,7 @@ public int rank(final Set<ResourceType> resourceTypes, final Collection<ServiceC
       }
 
       final Store.Provider candidateUnderlyingProvider = selectProvider(resourceTypes, serviceConfigs, xaServiceConfiguration);
-      return 10 + candidateUnderlyingProvider.rank(resourceTypes, serviceConfigs);
+      return 1000 + candidateUnderlyingProvider.rank(resourceTypes, serviceConfigs);
```

Comment:
No, this was on purpose -- to make room for `ClusteredStore`.  If my presumption about the implementation orderings needs to be changed, let me know.